src/index.ts

```diff
@@ -79,31 +84,95 @@ async function setDebugSession(
   app.commands.notifyCommandChanged();
 }
 
+function updateToolbar(
+  widget: NotebookPanel | ConsolePanel | FileEditor,
+  debug: IDebugger
+): void {
+  if (!debug.isDebuggingEnabled) {
+    return;
+  }
+  const isConsolePanel = widget instanceof ConsolePanel;
+
+  const changeAtribute = () => {
+    if (debug.session.debuggedClients.has(debug.session.client.name)) {
+      widget.node.setAttribute('data-jp-debugger', 'true');
+    } else {
+      widget.node.removeAttribute('data-jp-debugger');
+    }
+  };
+
+  const getToolbar = (): Toolbar => {
```

Comment:
Hm, aren't the three types of things that we support all main area widgets? They should all have a `.content` and a `.toolbar` so you shouldn't have to write a function to get the toolbar yourself.

Reply:
Only Console widget don't have toolbar and content. DocumentWidget and NotebookPanel have a toolbar. That's why I need a function for check if toolbar exists.

![console](https://user-images.githubusercontent.com/34033913/70720511-35e4b380-1cf4-11ea-96e0-2758362a6fbf.gif)

Reply:
Hm, this seems like a shortcoming in JupyterLab. I think the console should probably be a main area widget as well, for exactly this reason.

Reply:
cf. https://github.com/jupyterlab/jupyterlab/issues/7624

Reply:
Thanks @afshin for opening this issue.

For now we can use the current workaround, and open an issue in this repo to update the code later.